### PR TITLE
docs: add to docs after run-through

### PR DESCRIPTION
**Pull request by fernandoataoldotcom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-compute.googleapis.com
-```
 
-2. Service Quota increase: `In-use IP addresses (default is 8)`. Have this increased to 64. \_Note: this increase should be immediate, otherwise contact GCP support/account maangement\_
+2. Service Quota increase: `In-use IP addresses (default is 8)` has too low of a default value for deploying Kubernetes and quota should be increased to 64 to create headroom in available IPs.
+Edit the quota for `In-use IP addresses` in the [quota page](https://console.cloud.google.com/iam-admin/quotas) or with the following `gcloud` command.
+
+```bash
+gcloud alpha services quota update \
+    --service=compute.googleapis.com --consumer=projects/antoniostacos-gcp-nonprod \
```
Comment (venkatamutyala) [inline README.md:21]: should we make `antoniostacos-gcp-nonprod` a more explicit placeholder value?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
+    --unit=1/{project} --value=64
+```
+\_Note: If the increase isn't immediate or you receive an error, contact GCP support/account management\_
 
 ![Screenshot of Quota Increase](https://user-images.githubusercontent.com/6570292/210277244-f3a75d06-763f-4bdc-805e-4f8bd3c77ad5.png)
 
 ### Service Account
 
-1. Create a service account and download the json key file, save the file as `creds.json`
+1. [Create a service account](https://console.cloud.google.com/iam-admin/serviceaccounts/create) and [download the json key file](https://console.cloud.google.com/iam-admin/serviceaccounts/details/101612329871957262389/keys), save the file as `creds.json`
```
Comment (venkatamutyala) [inline README.md:31]: If the plan is to have them auth with gcloud then we could probably have them run the commands to create this with the gcloud cli too

File: README.md [hunk 3]
```diff
@@ -29,9 +37,14 @@ compute.googleapis.com
     export GOOGLE_CREDENTIALS=$(pwd)/creds.json
     ```
 
-3. In the project, give the service account the `Owner` role.
+3. In the project, [grant the service account the `Owner` role](https://console.cloud.google.com/iam-admin/iam) by selecting `GRANT ACCESS` at the top of the screen and filling out the principal (your service account role) and the `Owner` role.
```
Comment (venkatamutyala) [inline README.md:40]: See last comment about using the gcloud cli for this

File: README.md [hunk 4]
```diff
@@ -29,9 +37,14 @@ compute.googleapis.com
...
+
+## Terraform Deployment
 
-4. Example configuration for this module:
+### Configuration
+
+Create a `captain_configuration.tfvars` configuration file to deploy Kubernetes.  
+A reasonable xample configuration for this module, be sure to update `project_id`, required, and `region`, if desired:
```
Comment (venkatamutyala) [inline README.md:47]: `project_id` is definitely gcp specific

**Review by venkatamutyala — CHANGES_REQUESTED** [review]
@fernandoataoldotcom I left comments on the README.md but I know it should have been on the .header.md

File: README.md (hunk 1 above)
Comment (fernandoataoldotcom) [reply README.md:21]: great catch - I missed that one.

File: README.md (hunk 2 above)
Comment (venkatamutyala) [reply README.md:31]: created issue: https://github.com/GlueOps/terraform-module-cloud-gcp-kubernetes-cluster/issues/14

**Review by venkatamutyala — APPROVED** [review]

**Pull request merged**